content-src/experiments/snooze-tabs.yaml

```diff
@@ -96,4 +96,7 @@ contributors:
     title: 'Softvision QA'
     avatar: /static/images/experiments/avatars/ciprian.jpg
 created: '2017-02-22T17:00:00.00Z'
+graduation_url: http://example.com/graduation-report
```

Comment:
Do we want the fake `graduation_url` here (it isn't in **cliqz**, **min vid**, or **voice fill**.